README.md

```diff
@@ -275,12 +274,29 @@ This waiting period allows time for community comment and review, and to provide
 If there is some urgent need for the package to be registered (for example, an upcoming talk featuring the package or other external deadline), the waiting period can be expedited by request made in the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) channel on the 
 public Julia Language Slack, or directly on the PR itself. This request should include a link to the registration PR and why an expedited merge is needed. In this case, a registry maintainer (see below) can manually review the package (often with stricter standards than the typical package faces, given the lack of bandwidth to review all packages thoroughly).
 
-#### Who can approve an early merge?
+### Who can approve an early merge?
 
 Any registry maintainer may merge a package registration PR early.
 Early merges should still be discussed on the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) slack channel before merging.
 Registry maintainers are discouraged from merging packages they directly *or* indirectly maintain as mentioned in [CONTRIBUTING.md](https://github.com/JuliaRegistries/General/blob/master/CONTRIBUTING.md#other-ways-to-help).
 
+### When is yanking a release appropriate?
+
+Releases that have already merged can be marked as "yanked" by adding `yanked = true` under the release entry in `Versions.toml`. This will mean
+that Pkg will no longer consider them to exist and not resolve those versions, but that version cannot be re-registed in General.
+
+The primary reason yanking exists is to make a release impossible to install. This is intended to be used for circumstances when having this
```

Comment:
```suggestion
The reason yanking exists is to make a release impossible to install. This is intended to be used for circumstances when having this
```
Are there any other reasons?

Reply:
Yes, the second reason is because the package manager generally will chose to use an older patch release if that is what is required for compatibility with something else.
Which is a problem if the bug in the older patch release is incorrect compat bounds.

(I have argued that for this reason Pkg shouldn't actually install older patch releases without a special flag, but people disagreed, so yanking is often required when your bug is with compat bounds)

Reply:
I think of the compat reason as a subset of "make a release impossible to install."

Reply:
Though I guess the "primary" here refers to the following sentence, not the sentence it is a part of?

Reply:
```suggestion
The reason yanking exists is to make a release impossible to install. This is primary intended to be used for circumstances when having this
```

Reply:
we should add this, plus a footote aon "install" saying

```
To be precise it will still be possible to install if using the same manifest.
However any new manifest, such as those created by `Pkg.test` or `] up SomeUnrelatedPackage`, or `resolve` after changing julia version, will fail.
```

This will address @MasonProtter 's comment.